ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -0,0 +1,289 @@
+SITE_HOST: 'enikshay.in'
+#CAS_SITE_HOST: 'cas.commcarehq.org'
+#ENIKSHAY_SITE_HOST: 'enikshay.commcarehq.org'
+J2ME_SITE_HOST: 'j2me-enikshay.in'
+deploy_env: enikshay
+#tableau_server: 'tableau2.internal.commcarehq.org'
+#TABLEAU_HOST: 'icds.commcarehq.org'
+internal_domain_name: 'enikshay.in'
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: False
+  cchq_http: True
+  cchq_http_redirect: False
+  cchq_http_j2me: True
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech2: False
+  riakcs: True
+#  cas_ssl: True
+#  enikshay_ssl: True
+  couchdb2: False
+
+riak_ring_size: 64
```

Comment:
Also I don't know if you guys have already deployed riak but you should check:

```
root@pg0:~# riak-admin status | grep ring
ring_creation_size : 128
ring_members : ['nodes']
ring_num_partitions : 128
```

If num_partitions is 64 you should recreate the cluster asap